static/js/server_events_dispatch.js

```diff
@@ -539,7 +539,17 @@ export function dispatch_normal_event(event) {
             }
             break;
 
-        case "update_display_settings": {
+        case "user_settings": {
+            if (settings_config.all_notification_settings.includes(event.setting_name)) {
```

Comment:
How did you verify that `all_notification_settings` lists all notification setting event types?

Reply:
I checked that the settings in `all_notification_settings` and in `notification_setting_types` dict, except the `presence_enabled` and `email_notifications_batching_period_seconds` which is only in the `notification_setting_types` dict and we don't have any live update code for `presence_enabled` and `email_notifications_batching_period_seconds` is implemented in backend only.

Reply:
OK, makes sense. I expect it's a bug that `presence_enabled` does not have live-update code, hrm.